packages/jest-leak-detector/src/__tests__/index.test.js

```diff
@@ -11,6 +11,9 @@ afterEach(() => {
 });
 
 it('complains if the value is a primitive', () => {
+  expect(() => new LeakDetector(Symbol())).toThrowErrorMatchingSnapshot();
```

Comment:
mind putting these under the old `toThrowErrorMatchingSnapshot` assertions to minimize the diff in the snapshot?